frontend/public/scss/home-page/home-page-video-component.scss

```diff
@@ -21,36 +20,40 @@
   }
 
   .home-page-video-container {
-    margin-top: -60px;
+    margin-top: 60px;
     width: 100%;
-    max-width: 1440px;
+    max-width: 1320px;
     flex-shrink: 0;
     display: flex;
-    @include media-breakpoint-down(lg) {
+    padding: 0;
+    flex-wrap: wrap;
+    align-items: center;
+    justify-content: space-between;
+    @include media-breakpoint-down(xl) {
       flex-direction: column;
       width: 100%;
```

Comment:
Can this be removed since it's defined under `.home-page-video-container`?

Reply:
It's no longer 100 at home page video container and this confused me more than I should admit.